Frondgle/thearchive-server
# Cloud delete required fields elf

Ok, so I've tested and it works on my end. I changed the Art model slightly from Aj's previous branch, and I made the 'title', 'date_created', and 'tags' fields not required.

Recommended steps for testing:

1.  Fetch and checkout branch
2. Rerun migrations (Seed file)
3. Create Super User
4. Login to the Django dashboard
5. Add a file leaving title, date_created, and tags fields blank
6. Check Cloudinary Media Library to verify upload.
7. Delete file from Django Dashboard.
8. Verify file deletion in Cloudinary Media Library


Anytime changes are made to the Models, it is recommended to rerun steps 2-8. I will work on writing a Test for this.
